### PR TITLE
Support for Huawei VRP logs

### PR DESCRIPTION
**Pull request by inetAnt**
Hello,

This PR aims at adding support for Huawei switches logs.
Type of messages supported are INTERFACE_UP and INTERFACE_DOWN.

Thanks to @mirceaulinic for the help!

### CODE REVIEW

File: napalm_logs/config/huawei/init.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+prefixes:
+  - time_format: "%Y-%m-%d %H:%M:%S"
+    values:
+      date: (\d+-\d+-\d+)
+      time: (\d+:\d+:\d+)
+      milliseconds: (\d{3})
+      host: ([^ ]+)
+      tag: (\w+\/\w+\/\w+)
+      otherThings: (\(\w+\))
```
Comment (mirceaulinic) [inline napalm_logs/config/huawei/init.yml]: LOL you really let it like that... I'd suggest renaming this to something else please. :D

Comment (inetAnt) [reply napalm_logs/config/huawei/init.yml]: Done :grin: (I used http://support.huawei.com/enterprise/en/doc/EDOC0100533434)

Comment (mirceaulinic) [reply napalm_logs/config/huawei/init.yml]: Makes sense, thanks!

**Comment by loverend** [comment]
Looks great. Thanks very much.

**Pull request merged**